instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/auto/instrumentation/netty/v4_1/client/HttpClientRequestTracingHandler.java

```diff
@@ -59,29 +53,21 @@ public void write(final ChannelHandlerContext ctx, final Object msg, final Chann
       ctx.channel().attr(AttributeKeys.CLIENT_PARENT_ATTRIBUTE_KEY).set(null);
     }
 
-    Span span =
-        TRACER.spanBuilder(DECORATE.spanNameForRequest(request)).setSpanKind(CLIENT).startSpan();
-    try (Scope scope = TRACER.withSpan(span)) {
-      DECORATE.afterStart(span);
-      DECORATE.onRequest(span, request);
-      DECORATE.onPeerConnection(span, (InetSocketAddress) ctx.channel().remoteAddress());
-
+    Span span = TRACER.startSpan(request);
+    TRACER.onPeerConnection(span, (InetSocketAddress) ctx.channel().remoteAddress());
+    Scope currentScope = null;
+    try (Scope scope = currentContextWith(span)) {
       // AWS calls are often signed, so we can't add headers without breaking the signature.
       if (!request.headers().contains("amz-sdk-invocation-id")) {
-        Context context = withSpan(span, Context.current());
-        OpenTelemetry.getPropagators()
-            .getHttpTextFormat()
-            .inject(context, request.headers(), SETTER);
+        currentScope = TRACER.startScope(span, request);
```

Comment:
scope is already started with this span above, i think don't use `startScope` here